boa_cli/src/helper.rs

```diff
@@ -139,52 +139,73 @@ static KEYWORDS: Set<&'static str> = phf_set! {
     "let",
 };
 
-struct LineHighlighter;
-
-impl Highlighter for LineHighlighter {
-    fn highlight<'l>(&self, line: &'l str, _: usize) -> Cow<'l, str> {
-        let mut coloured = line.to_string();
+struct LineHighlighter {
+    regex: Regex,
+}
 
-        let reg = Regex::new(
+impl LineHighlighter {
+    fn new() -> Self {
+        // Precompiles the regex to avoid creating it again after every highlight
+        let regex = Regex::new(
```

Comment:
Wouldn't this be faster if we used `once_cell` with a static variable? It would only need to be compiled once for the whole execution (or even for multiple executions)

Reply:
I don't think it helps in this case, because `RLHelper` is only initialized once before entering an infinite loop, so it essentially acts as a `OnceCell` already.

Reply:
Sounds good, yes :) LGTM!